app/helpers/date_time_helper.rb

```diff
@@ -44,4 +44,13 @@ def api_date_format(date_time)
       end
     end
   end
+
+  def to_be_debated_on_in_words(date)
+    return unless date.present?
```

Comment:
By doing the checking for the debate date here you end up with and empty `<p>` tag which is undesirable - we should use `scheduled_debate_date?` on the petition in the view and remove this check. E.g:

``` ruby
<% if petition.scheduled_debate_date? %>
<p><%= to_be_debated_on_in_words(petition.scheduled_debate_date) %></p>
<% end %>
```